sdk/storage/azblob/blob/models.go

```diff
@@ -34,11 +34,11 @@ type LeaseAccessConditions = exported.LeaseAccessConditions
 // ModifiedAccessConditions contains a group of parameters for specifying access conditions.
 type ModifiedAccessConditions = exported.ModifiedAccessConditions
 
-// CpkInfo contains a group of parameters for client provided encryption key.
-type CpkInfo = generated.CpkInfo
+// CPKInfo contains a group of parameters for client provided encryption key.
+type CPKInfo = generated.CpkInfo
```

Comment:
I think the generated content needs to be transformed too in case there are exported fields with the pascal-cased names.

Reply:
Should that change be on the autorest side as well? if this a rule we want to follow across GO, in the meantime I can introduce a transform

Reply:
Changes applied to generated code.